src/map/unit.cpp

```diff
@@ -75,6 +75,68 @@ static TIMER_FUNC(unit_attack_timer);
 static TIMER_FUNC(unit_walktoxy_timer);
 int32 unit_unattackable(struct block_list *bl);
 
+void view_data::update( map_session_data& sd, _look look ){
+	int32 val = sd.vd.look[look];
+
+	switch( look ){
+		case LOOK_WEAPON:
```

Comment:
These two statements look the same, except for the 2h sword/shield check. Can we just keep them together?

Reply:
Keep them together?
They were separated before in clif.cpp too.
But we can think about mergint them, if they stay readable.
Do you have a good suggestion for merging the two cases together?

Reply:
I would prefer to keep it separate (like it was).
With this suggestion we make the code less readable and we have to double check a few ifs just to prevent a few lines.